src/main.ts

```diff
@@ -1,8 +1,19 @@
 import { NestFactory } from '@nestjs/core';
+import { DocumentBuilder, SwaggerModule } from '@nestjs/swagger';
 import { AppModule } from './app.module';
 
 async function bootstrap() {
   const app = await NestFactory.create(AppModule);
+
+  const SwaggerConfig = new DocumentBuilder()
+    .setTitle('Swyft API')
+    .setDescription('The Swyft banking API')
+    .setVersion('1.0')
+    .addTag('payment')
```

Comment:
If you do the swagger changes in the controllers you can delete this line: 
.addTag('payment')